### PR TITLE
chore: pull in redis/pub-sub/kvs/ttk/state plugin/state machine code for upcoming flows

### PR DESCRIPTION
**Pull request by kleyow**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -18,6 +18,9 @@ services:
       - mojaloop-net
     depends_on:
       - mysql
+      - redis
+    extra_hosts:
```
Comment (lewisdaly) [inline docker-compose.yml:22]: Why do we need `extra_hosts`?

File: docker-compose.yml [hunk 2]
```diff
@@ -46,4 +49,51 @@ services:
...
+      - "./docker/ml-testing-toolkit/secrets:/opt/mojaloop-testing-toolkit/secrets"
+    ports:
+      - "15000:5000"
+      - "5050:5050"
+    command: npm start
+    networks:
+      - mojaloop-net
+    depends_on:
+      - mongo
+    extra_hosts:
```
Comment (lewisdaly) [inline docker-compose.yml:77]: Same as above. We don't need `extra _hosts`, it just understanding the networking more difficult.

**Review by lewisdaly — CHANGES_REQUESTED** [review]
Looks good, just a minor comment on `extra_hosts`

File: docker-compose.yml (hunk 1 above)
Comment (kleyow) [reply docker-compose.yml]: Don't believe we need it. Just mindless copying. 
Let me know if I set up the compose files properly like the pisp repo.

Comment (lewisdaly) [reply docker-compose.yml]: Cool. Yeah it looks fine to me apart from that section. We'll know if the config is wrong because the tests will break.

**Review by lewisdaly — APPROVED** [review]

**Pull request merged**